tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionEntity.cs

```diff
@@ -33,6 +37,28 @@ public sealed class SolutionEntity : EntityBase
         public string WorkOfPlan { get; set; }
 
         protected override string InsertSql => @"
+
+        IF EXISTS(select * from dbo.Solution where Id=@SolutionId)
+            update dbo.Solution
+                 
+            SET   Version=@Version,
+                Summary=@Summary,
+                FullDescription=@FullDescription,
+                Features=@Features,
+                ClientApplication=@ClientApplication,
+                Hosting=@Hosting,
+                RoadMap=@RoadMap,
+                IntegrationsUrl=@IntegrationsUrl,
+                AboutUrl=@AboutUrl,
+                ServiceLevelAgreement=@ServiceLevelAgreement,
+                WorkOfPlan=@WorkOfPlan,
+                ImplementationDetail=@ImplementationDetail,
+                LastUpdated=@LastUpdated,
+                LastUpdatedBy=@LastUpdatedBy
+
+                where Id=@SolutionId
+        ELSE
+
```

Comment:
This addition shouldn't be necessary. The existing SQL should be sufficient.

Reply:
As part of merging query of Insert and Update of Solution, We have to update this query.